core/spec/services/spree/import_service/products/create_spec.rb

```
@@ -0,0 +1,80 @@
+        option1_name: create(:option_type).name,
+        option1_value: create(:option_value).name,
+        option2_name: create(:option_type).name,
+        option2_value: create(:option_value).name,
+        property1_name: create(:property).name,
+        property1_value: 'bebe',
+        property2_name: create(:property).name,
+        property2_value: 'qwer'
+      }
+    end
+
+    let(:create_klass) { ImportService::Products::Create }
```

RuboCop (1.75.5)

[convention] Use `described_class` instead of `ImportService::Products::Create`.

(RSpec/DescribedClass)